src/js/components/members.js

```diff
@@ -0,0 +1,37 @@
+import { newMultimedia, newContainer, newText } from './helpers';
+
+const createElementsMembers = (members) => {
+  return members.map((member) => {
+    const membersText = newText('h3', member.name, ['heading', 'heading-3']);
+    const membersProfession = newText('p', member.job, [
+      'text',
+      'text-largue',
```

Comment:
No sure but is "Large" instead of "Largue" or what do you mean?

Reply:
Done